apps/zotonic_mod_wires/priv/lib/js/apps/zotonic-wired.js

```diff
@@ -893,13 +893,9 @@ function z_typeselect(ElementId, postbackInfo)
 
     z_input_updater = setTimeout(function()
     {
-        var obj = $('#'+ElementId);
-
-        if(obj.val().length >= 2)
```

Comment:
Maybe make it >= 1 ?

Then we don't post on 0 chars.

Reply:
But then it won't reset the search results when you go from 1 to 0, which is what users expect.

Reply:
Should we return the empty result for the empty query?  Or just anything that matches?

Reply:
The example code in the documentation is also interesting. A commented out wire statement.

<img width="959" alt="Screen Shot 2022-11-28 at 12 33 03" src="https://user-images.githubusercontent.com/1024972/204267722-6601a0be-395f-4cb5-89b6-2d85a21502f5.png">

Reply:
Side issue.. the color scheme for example code on zotonic.com makes it look like wire statements are comments.

<img width="717" alt="Screen Shot 2022-11-28 at 12 38 03" src="https://user-images.githubusercontent.com/1024972/204268589-3c22e8b0-7027-4cc6-93c6-f6f6c0e4c35e.png">

Reply:
Ah.. `%`

Reply:
The typeaction uses the autocomplete query. This returns 0 result without querying the database. 

I've updated the documentation to correctly use an input type="search" input box. This makes sure a "Search" button appears on the keyboard of an iOS device. On Safari an Chrome the search box can also be cleared by typing <kbd>Esc</kbd> or clicking on a small cross appearing at the right of the input box. For the clickable cross the type of the wire must be changed from `keyup` to `input`.